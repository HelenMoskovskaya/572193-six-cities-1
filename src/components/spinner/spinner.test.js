import React from 'react';
import renderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router';
import Spinner from './spinner.jsx';


describe(`Spinner`, () => {
  it(`renders component correctly`, () => {
    const tree = renderer.
      create(<MemoryRouter>
        <Spinner
        />
      </MemoryRouter>).
      toJSON();

    expect(tree).toMatchSnapshot();
  });
});
