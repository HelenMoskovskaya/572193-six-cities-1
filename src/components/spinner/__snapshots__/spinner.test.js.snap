// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders component correctly 1`] = `
<div
  className="cssload-container"
>
  <div
    className="cssload-loading"
  >
    <i />
    <i />
  </div>
</div>
`;
